src/magic_folder/status.py

```diff
@@ -38,7 +42,7 @@ class IStatus(Interface):
     messages from the status API.
     """
 
-    def error_occurred(public_error):
+    def error_occurred(folder, message):
```

Comment:
Hmm .. although I suppose _most_ errors will be "for a folder" I imagine there are cases of more generic errors we'll want to report. Forcing this API to have a "folder" arg just to do the wrapper-thing "more generically" feels pretty forced.  (Or is it just the interface here that's already-wrong since the args seem to refer to the "exception" version).

Anyway, "burning in" that all errors only occur in the context of a single magic-folder doesn't feel correct.

Reply:
This is just fixing up the interface to match the implementation.

Reply:
As an aside, I agree that we should be able to report errors that aren't tied to a folder; I think when I was writing this, one of the reasons I had in my head for filtering methods to proxy was being able to report global errors.